app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -248,8 +248,16 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
         // code paths consistent between the two.
         if (event.keyCode == KeyEvent.KEYCODE_BACK ||
                 event.keyCode == KeyEvent.KEYCODE_DEL) return super.dispatchKeyEvent(event)
+
         serviceLocator.cursorEventRepo.pushKeyEvent(event)
```

Comment:
Is there any reason not to move this call inside `cursorModel.handleKeyEvent()` so that we can have a single entry point to cursor key code?

Reply:
Hm, looks like this forces a cyclical dependency between `CursorModel` and `CursorEventRepo`.  I don't think this change is worth that, but let me know if you disagree.